packages/config/src/projects/facet/facet.ts

```diff
@@ -39,55 +83,153 @@ export const facet: ScalingProject = opStackL2({
       ],
     },
   },
-  nonTemplateEscrows: [
-    discovery.getEscrowDetails({
-      address: EthereumAddress('0x0000000000000b07ED001607f5263D85bf28Ce4C'),
-      tokens: ['ETH'],
-      description: 'Fast bridge contract.',
-    }),
-  ],
-  nonTemplateRiskView: {
-    stateValidation: RISK_VIEW.STATE_NONE,
-    dataAvailability: RISK_VIEW.DATA_ON_CHAIN,
-    exitWindow: RISK_VIEW.EXIT_WINDOW(0, FINALIZATION_PERIOD_SECONDS),
-    sequencerFailure: RISK_VIEW.SEQUENCER_SELF_SEQUENCE_NO_SEQUENCER,
-    proposerFailure: RISK_VIEW.PROPOSER_CANNOT_WITHDRAW,
+  config: {
+    escrows: [
+      discovery.getEscrowDetails({
+        address: EthereumAddress('0x0000000000000b07ED001607f5263D85bf28Ce4C'),
+        tokens: ['ETH'],
+        description: 'Fast bridge contract.',
+      }),
+      discovery.getEscrowDetails({
+        address: EthereumAddress('0x79009BE75Ef97e17b2832CE349a1CE7e60dac262'),
+        tokens: ['ETH'],
+        description: 'Escrow for ETH bridge.',
+      }),
+      discovery.getEscrowDetails({
+        address: EthereumAddress('0x8F75466D69a52EF53C7363F38834bEfC027A2909'),
+        tokens: ['ETH', 'WETH'],
+        description: 'L1StandardBridge (deprecated).',
+      }),
+    ],
+  },
+  riskView: {
+    stateValidation: {
+      ...RISK_VIEW.STATE_ZKP_OPTIMISTIC,
+      secondLine: formatExecutionDelay(MAX_CHALLENGE_SECS),
+    },
+    dataAvailability: {
+      ...DATA_ON_CHAIN,
+    },
+    exitWindow: {
+      value: 'Immutable',
```

Comment:
exit windows are not "immutable", contracts are, so it sounds a bit off here. see how fuel v1 was handled